hedera-node/src/main/java/com/hedera/services/files/HederaFs.java

```diff
@@ -64,6 +57,12 @@
 	 */
 	FileID create(byte[] contents, HFileMeta attr, AccountID sponsor);
 
+	void sudoSetattr(FileID id, HFileMeta attr);
+	void setattr(FileID id, HFileMeta attr);
+	void overwrite(FileID id, byte[] newContents);
+	void append(FileID id, byte[] moreContents);
+	void delete(FileID id);
+
```

Comment:
I think we still need docs for those methods

Reply:
Aren't these automatically generated at some point of the process of Hashgraph?

Reply:
Nope, we have to write them down